metadata.tvshows.thetvdb.com.v4.python/resources/lib/actions_test.py

```diff
@@ -0,0 +1,13 @@
+import unittest
```

Comment:
You probably didn't mean to upstream this?

Reply:
The tests were created before me. I didn't remove them because they are helpful in some case and maintaining two versions of code - with and without tests - adds more effort. Personally, I don't see any harm to have 2 additional files in the distribution.

Reply:
I usually exclude them from the upload via gitattributes https://github.com/trakt/script.trakt/blob/main/.gitattributes as the uploader uses these

Reply:
Not sure that it'll help in my case. I've removed the files manually anyway.